### ISSUE TITLE
Spectacle won't work on jetbrains IDE's.

### ISSUE THREAD

**Issue by Neener54** [eczarny/spectacle#500]
I recently updated my jetbrains IDE's (webstorm, rubymine, phpstorm, etc) and now none of them will respond to spectacle. I tried deleting all of the config files and starting over in Intellij in case I had messed up a setting the the apps, didn't fix it. I reinstalled spectacle hoping that would help.

Oddly, every other application works just fine. I would assume that it's something Jetbrains is doing and not really related to spectacle. I'm going to submit a ticket in their system as well.

**Comment by hankwallace** [comment]
After my computer wakes from sleep, the IDE's don't respond to Spectacle anymore. At least for me, restarting the IDE fixes it.

**Comment by brian-vogogo** [comment]
@Neener54 do you have a reference to the Jetbrains bug/post? I am having the same issue with RubyMine.

**Comment by l8on** [comment]
I'm having the same issue with WebStorm. Was working quite well until I updated to WebStorm 11.0.2
@Neener54 I'd also like a link to that issue so I can add some volume to our collective complaint.

**Comment by Neener54** [comment]
https://youtrack.jetbrains.com/issue/IDEA-149718 This is the issue number for @brian-vogogo @l8on Per @hankwallace it does work if you restart the IDE. It appears to stop working on sleep and possibly when the screensaver starts.

**Comment by oafridi** [comment]
yup same issue here. Any workarounds?

**Comment by eczarny** [comment]
When I used IntelliJ IDEA I ran into this problem periodically as well. Restarting the IDE usually worked for me. I'm inclined to close this issue in favor of https://youtrack.jetbrains.com/issue/IDEA-149718. If the folks at JetBrains are interested I can offer some help debugging the issue.

I do know that IntelliJ IDEA would only work with Java 6 for a very long time. I'm wondering if this is just a problem with the migration away from Apple's Java 6 distribution.
